oam/install_scripts/mcs-loadbrm.py

```diff
@@ -46,17 +48,17 @@
                 pass
         except subprocess.CalledProcessError as e:
             # will happen when brm file does not exist
-            pass
+            print(f'{brm} does not exist.', file=sys.stderr)
     else:
         # load local dbrm
         try:
             brm_saves_current = subprocess.check_output(['cat', brm])
         except subprocess.CalledProcessError as e:
             # will happen when brm file does not exist
-            pass
+            print(f'{brm} does not exist.', file=sys.stderr)
 
 if brm_saves_current:
-    cmd = loadbrm.format(brm_saves_current.decode('utf-8'))
+    cmd = f'{loadbrm} {dbrmroot}{brm_saves_current.decode("utf-8").replace("BRM_saves", "")}'
```

Comment:
Both dbrmroot and brm_saves_current variable may contain BRM_saves string so let's replace it in brm_saves_current